frontend/src/upload.js

```diff
@@ -205,28 +204,32 @@ $(document).ready(function() {
     fileSender.on('loading', isStillLoading => {
       // The file is loading into Firefox at this stage
       if (isStillLoading) {
-        console.log('Processing');
-      } else {
-        console.log('Finished processing');
```

Comment:
I killed these "finished" messages because they appear super briefly, plus I didn't want to bother our excellent l10n community.